src/Locale/it_IT/default.po

```diff
@@ -512,6 +548,18 @@ msgstr "mostra file"
 msgid "www.bedita.com"
 msgstr ""
 
+#: src/Template/Layout/js/app/app.js:138
+msgid "Please insert a new title on \"${ title }\" clone"
```

Comment:
i'd expected standard placeholders like:
``` 
msgid "Please insert a new title on \"{0}\" clone"
```

Reply:
ttag syntax is different, as explained in ttag doc page https://github.com/ttag-org/ttag